docs/Datagrid.md

```diff
@@ -900,6 +900,8 @@ const PostList = () => (
 );
 ```
 
+**Tip:** You may need to clear your local storage to reflect the changes, as react-admin saves the computed column names in the Store.
```

Comment:
Might be worth linking to store upgrade (versionning) doc

Reply:
I disagree, to me this tip is targeting the developer, not the end user

Reply:
@fzaninotto shares your opinion, so I'll make the change.
He also noted that there already exists a wrapper component, so I used it instead in the code examples.